### DISCUSSION TITLE
Refining Existing Models

### DISCUSSION THREAD

**Discussion by ADHDSquir** [dscripka/openWakeWord#225] [Q&A]
I trained a model a few months ago that is working decently well. Since starting to use the model, I have collected thousands of samples of false positives and false negatives from my real-world environment using my production hardware. Is it possible to use these samples to further train my model or do I have to build a new model from scratch and supplement the generated samples with my data?

**Comment by sangheonEN** [comment]
This is not an answer to your question, but could you give me some advice on how you trained your custom wake word model?

1. How did you build your training dataset?
ex) How did you build your positive samples? Did you record them yourself? Did you augment the data? If you augmented the data, how did you do it? Is there an open data augmentation code?

2. Training method
ex) What training code did you use? How did you set the parameters?

I think the training model is performing too poorly because I have very few positive samples. This is explained in "https://github.com/dscripka/openWakeWord/discussions/227". Could you please help me?
